hack/jenkins/e2e-runner.sh

```diff
@@ -192,13 +192,18 @@ function e2e_test() {
         echo "Publish version to ci/latest-green.txt: $(cat version)"
         gsutil cp ./version gs://kubernetes-release/ci/latest-green.txt
     fi
+    return ${exitcode}
 }
 
 echo "--------------------------------------------------------------------------------"
 echo "Test Environment:"
 printenv | sort
 echo "--------------------------------------------------------------------------------"
 
+# Set this var instead of exiting-- we must do the cluster teardown step. We'll
+# return this at the very end.
+EXIT_CODE=0
```

Comment:
Yeah I dunno what the rules are in shell, but I like my local variables and global ones to not interact...